src/uct/cuda/cuda_copy/cuda_copy_md.c

```diff
@@ -50,6 +50,11 @@ static ucs_config_field_t uct_cuda_copy_md_config_table[] = {
      " is registered. Otherwise only the user specified region is registered.",
      ucs_offsetof(uct_cuda_copy_md_config_t, max_reg_ratio), UCS_CONFIG_TYPE_DOUBLE},
 
+    {"DMABUF_ENABLE", "try",
```

Comment:
"DMABUF"

Reply:
it is inconsistent with our other vars which enable something (hwtm, cqe zipping, etc)